### DISCUSSION TITLE
New build v31 - ETA?

### DISCUSSION THREAD

**Discussion by eviewer** [dlemstra/magick-wasm#179] [General]
Hello do we know when the next build is planned to be released? Are there any build instructions to setup build environment?

**Comment by dlemstra** [comment]
I don't have a plan or eta when that will happen. This weekend we will probably do an ImageMagick release and I could do a release of this library afterwards.

There is a contributing.md file that explains what you need to do to make `npm install` work. There are no other docs about build setup but this project does contain a devcontainer file. Another option would be looking at the `package.json` and `main.yml` build file to figure out how this project should be build.

**Comment by dlemstra** [comment]
The new release has been published.
